### ISSUE TITLE
Checkbox Group - Validation Not Working

### ISSUE THREAD

**Issue by chris-scholl** [victorjonsson/jQuery-Form-Validator#591]
I have submitted this previously (#582) but it was closed due to some issues with my examples.  Hopefully you can help with this issue.

When validating a checkbox group the documentation reads:
Validate qty of checkboxes in a group (same name) have been checked, using min, max or range. Only the first checkbox element in the group needs to have the validation attributes added.

If the validation attributes are not on ALL checkboxes the validation fails.

Here is an example of the behavior, the validation works when checking and unchecking the first checkbox. When you try to check and uncheck any other checkbox in the group, the validation does not occur.

[http://codepen.io/chris-scholl/pen/eWmOpg](url)

I was adding the validation attributes to all checkboxes as a work-around but this causes the form to slow down and lock up the browser for 5-30 seconds at a time.

The main issue is that the validation only occurs on the checkbox array when the first checkbox is checked or unchecked.

One example is this:

- check the first checkbox
- check the second checkbox
- uncheck the first checkbox
- unchecking the second checkbox does not appear to remove the valid (green) state on the checkbox array

Another example is just checking any box other than the first, nothing appears to happen.

Thank you for taking the time to look into this issue.

**Comment by victorjonsson** [comment]
This is a bug in the module `toggleDisabled`. I guess you could say that `toggleDisabled` does not support the checkbox-group validator. The checkbox-group validator, however, works as it's supposed to, as you can see here http://codepen.io/anon/pen/RVxNyz. 

I will try to have this fixed asap.

**Comment by chris-scholl** [comment]
Even when the toggleDisabled module is not in play the checkbox validation does not appear to work properly.  If you visit http://codepen.io/anon/pen/RVxNyz and check any checkbox but the first, the validation does not fire.

**Comment by victorjonsson** [comment]
That is the expected behaviour. 

You don't display an answer as invalid before the user has completed his/her answer. The detection of a completed answer is often that the user leaves the input field and starts typing in another input field. In this case, when it's a number of checkboxes that should be ticked, we can't display the error message until the submit button gets pressed.

**Comment by chris-scholl** [comment]
It sounds as if I will not be able to use the toggleDisabled module to disable the submit button with checkbox arrays if the only way it validates properly is on submit.  Is that correct?

**Comment by victorjonsson** [comment]
That is correct. The module `toggleDisabled` does currently not support the checkbox-group validator

**Comment by victorjonsson** [comment]
This should now be fixed in 2.3.62
